keps/sig-node/4622-topologymanager-max-allowable-numa-nodes/README.md

```diff
@@ -334,24 +332,25 @@ well as the [existing list] of feature gates.
 [existing list]: https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/
 -->
 
-- [x] Feature gate (also fill in values in `kep.yaml`)
-  - Feature gate name: 
+- [X] Feature gate (also fill in values in `kep.yaml`)
+  - Feature gate name:
     - `TopologyManagerPolicyBetaOptions`
```

Comment:
@ffromani what should one do here for GA policies with TopologyManager?

I take it that this is no longer toggleable via beta options?

Reply:
I think we should remove `TopologyManagerPolicyBetaOptions `, leaving only `TopologyManagerPolicyOptions`